src/main/CommandLine.cpp

```diff
@@ -797,6 +801,53 @@ runRebuildLedgerFromBuckets(CommandLineArgs const& args)
     });
 }
 
+int
+runSimulate(CommandLineArgs const& args)
+{
+    CommandLine::ConfigOption configOption;
+    uint32_t desiredOps = 0;
+    uint32_t startLedger = 0;
+    uint32_t stopLedger = 0;
+    std::string historyArchiveGet;
+    return runWithHelp(
+        args,
+        {configurationParser(configOption),
+         clara::Opt{desiredOps, "DESIRED-OPERATIONS"}["--desired-operations"](
+             "desired number of operations per ledger, will never be less but "
+             "could be up to to 100 more"),
+         clara::Opt{startLedger, "START-LEDGER"}["--start-ledger"](
+             "ledger to start reading from history archive"),
```

Comment:
start/stop doesn't really convey a range? maybe from/to instead? Also, those are inclusive bounds, right?
Alternatively, we could use the same syntax than catchup that is "start/count"

Reply:
ok